2022/wordle/german/sentence_translations.json

```diff
@@ -1541,7 +1610,7 @@
  {
   "input": "By the end of the simulation, the average score works out to be about 4.124.",
   "model": "nmt",
-  "translatedText": "Am Ende der Simulation liegt die durchschnittliche Punktzahl bei etwa 4.124.",
+  "translatedText": "Liegt die durchschnittliche Anzahl am Ende bei 4.124",
```

Comment:
How will we handle decimals?
"Vier-Punkt-Eins-Zwo-Vier" sounds more like some subsection of something rather than an actual decimal number like "vier-komma-eins-zwo-vier"

Reply:
> How will we handle decimals? "Vier-Punkt-Eins-Zwo-Vier" sounds more like some subsection of something rather than an actual decimal number like "vier-komma-eins-zwo-vier"

Very good point that slipped by attention. It's probably best to convert to ",".